server/tests/trip.test.js

```diff
@@ -200,4 +200,46 @@ describe('Trip test suite', () => {
         done(err);
       });
   });
+  it('should accept a trip request', (done) => {
+    router()
+      .patch('/api/v1/trips/1')
+      .set('token', managerToken)
+      .send({
+        status: 'accepted'
+      })
+      .end((err, res) => {
+        expect(res.body.status).to.equal(200);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('data');
+        done(err);
+      });
+  });
+  it('should not accept or reject a trip request as the status sent is wrong', (done) => {
+    router()
+      .patch('/api/v1/trips/1')
+      .set('token', managerToken)
+      .send({
+        status: 'no edit'
+      })
+      .end((err, res) => {
+        expect(res.body.status).to.equal(400);
+        expect(res.body).to.be.an('object');
+        expect(res.body).to.have.property('error');
+        done(err);
+      });
+  });
+  it('should not accept or reject to a trip request as the user is not allowed to update the trip and the trip doesn\'t exist', (done) => {
```

Comment:
You should be explicit. It is clear that the request will be rejected here. Thus, use **should reject the trip request when the user has no permission or the trip does not exist**.

Reply:
feedback implemented

Reply:
Not appropriately, remove the **Thus, use**. You have copy-pasted everything.

Reply:
Sorry for that!!